website/docs/components/form/input-group/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,114 @@
+## Usage
+
+An Input Group is intentionally published as an agnostic container element that wraps one or more inputs and actions, making the relevant use cases broad.
+
+!!! Info
+
+The way in which these components are published and assembled together are fundamenally different between the Figma Ember components.
+
+- In Figma, multiple primitive components are published (`ButtonType` and `InputType`) that are intended to be assembled in an auto layout container. A pre-assembled `Base` component is also available for more simple instances.
+- The Ember component acts as a container to enforce consistent styling and grouping of nested Helios form components. It does not handle any logic or functionality out of the box, this is left up to the nested form elements.
+!!!
+
+### When to use
+
+- When filtering complex data, searching within a set of filters, and creating complex filtering patterns.
+- When collecting information in a form that benefits from a "connected" visual representation, e.g., a key/value pair.
+- When an action performs a generative function rather than a submit function, e.g., generating API credentials.
+- As a complex data input, e.g., uploading a document or connecting to an external resource.
+
+### When not to use
+
+- To submit a form or set of inputs. An Input Group can exist within a form to collect multi-directional data, but should not submit that data via a form method. Use the guidelines in [Form patterns](/patterns/form-patterns) instead.
+- To connect multiple fields that each require their own label.
+
+## Primitive components
+
+An Input Group can consist of a multiple primitive subcomponents that vary in their properties and intended input value. We recommend limiting usage in both Figma and code to those defined in the `ButtonType` and `InputType` primitives to ensure consistent styling and grouping.
+
+### Button types
+
+- **Default**: a default button can perform a function or interaction within an Input Group, e.g., generating an object, string, or value in a field it is connected to. This is an extension of the Helios [Button](/components/button).
+- **Toggle**: a toggle button can be used to filter or narrow the available parameters when searching within a data set or group of objects. This is an extension of the Helios [Toggle Button](/components/toggle-button).
```

Comment:
there's no such link as `toggle-button`

Reply:
Indeed, this doesn't match what we have in code. There is a `<Hds::Dropdown::Toggle::Button />`, but it's not of much use (functionally) unless it's part of the dropdown component `<Hds::Dropdown as |dd|><dd.ToggleButton /></Hds::Dropdown>`. So I guess, from a technical point of view, it probably makes sense to split the two types as `<Hds::Button>` and `<Hds::Dropdown>`. I can give this a bit more thinking 🤔

Reply:
We refer to this as a `ToggleButton` in Figma, probably should discuss a bit more how much the semantics should overlap here.